internal/link/link.go

```diff
@@ -99,25 +99,26 @@ func LinkServices(ctx context.Context, projectRef, anonKey string, fsys afero.Fs
 			fmt.Fprintln(os.Stderr, err)
 		}
 	}()
-	api := tenant.NewTenantAPI(ctx, projectRef, anonKey)
-	go func() {
-		defer wg.Done()
-		if err := linkPostgrestVersion(ctx, api, fsys); err != nil && viper.GetBool("DEBUG") {
-			fmt.Fprintln(os.Stderr, err)
+	api := tenant.NewTenantAPI(projectRef, anonKey)
+	version, err := api.GetServiceVersions(ctx)
+	if err != nil {
+		fmt.Fprintln(utils.GetDebugLogger(), err)
```

Comment:
**question:**

Just FMI I wonder what's the recommended handling of errors in the cli codebase. Some are sent to `os.Stderr` some like here to `utils.GetDebugLogger` how do we choose between one another ?